ratlib/autocorrect.py

```diff
@@ -15,15 +15,16 @@
 
 class CorrectionResult:
     # Table of lookalikes
-    lookalikes = {'L': dict(zip("01258", "oizsb"))}
-    lookalikes['D'] = dict((v, k) for k, v in lookalikes['L'].items())
+    digits = "001258"
+    letters = "doizsb"
+    lookalikes = {'L': dict(zip(digits, letters)), 'D': dict(zip(letters, digits))}
```

Comment:
if we are going to change these lines around, we may as well use dictionary comprehensions.